clients/client/src/test/java/org/projectnessie/client/auth/TestBearerAuthFilter.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.client.auth;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Nit: use assertj (we use it everywhere else)

Reply:
Migrated to AssertJ in 4a3d18717ad8bc8d65bfc422bdb34a3c7abe5084